configure

```diff
@@ -2927,7 +2927,7 @@ $as_echo_n "checking for python interpreter for cross build... " >&6; }
 	    if $interp -c "import sys;sys.exit(not '.'.join(str(n) for n in sys.version_info[:2]) == '$PACKAGE_VERSION')"; then
 	        break
 	    fi
-            interp=
```

Comment:
I could restore this?  
But I'm not sure it's necessary? 
Do you automatically rewrite ./configure and check in ./configure.ac has changed, or do I need to supply it in my pull?
If I do need to provide an updated ./configure in the pull,  do I have to use a specific version of autotools ( I notice this one is a bit dated ?)